go/git/put_get_test.go

```diff
@@ -86,6 +88,9 @@ func TestPutAndGet(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 1, len(oneRepo), "expected to get only one repo back, found: %d", len(oneRepo))
 	require.Equal(t, "repoNameFirst", string(oneRepo[0].LocalMetadata.RepoName))
+	require.Equal(t, kbtest.DefaultDeviceName, oneRepo[0].ServerMetadata.LastModifyingDeviceName)
+	require.Equal(t, string(team1.Chain.Id+"_abc123"), oneRepo[0].GlobalUniqueID)
+	require.Equal(t, "keybase://team/"+teamName1+"/repoNameFirst", oneRepo[0].RepoUrl)
```

Comment:
this doesn't make sense for implicit teams, is that a problem?

Reply:
nevermind, it's fine

Reply:
This is generated from the repo name and the `Folder`, so for an implicit team we'd expect either `keybase://private/...` or `keybase://public/...`. Does that sound right?

Reply:
yes. Might be worth adding a `GetMetadata` to `TestPutAndGetImplicitTeam` below

Reply:
good call, adding this caught a bug in some old code that was assuming the `TEAM` type.